src/goGetImport.ts

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+import vscode = require('vscode');
+import cp = require('child_process');
+import path = require('path');
+import fs = require('fs');
+import { getGoRuntimePath } from './goPath';
+import { getCurrentGoPath, getImportPath } from './util';
+
+export function goGetImport() {
+	const editor = vscode.window.activeTextEditor;
+	const selection = editor.selection;
+	const selectedText = editor.document.lineAt(selection.active.line).text;
+
+	const importPath = getImportPath(selectedText);
```

Comment:
If user runs this command on a line thats not an import statement, shouldnt we exit earlier? We know `go get` would fail then.

Reply:
Yup absolutely. Forgot that check.